quasar-osgi-tests/kryo-serializers/build.gradle

```diff
@@ -0,0 +1,67 @@
+import static aQute.bnd.version.MavenVersion.parseMavenString
```

Comment:
Just to check - are you still requiring the version of this from `runtime-os`?

Reply:
I guess my question is more do you need it here given it's also in runtime-os? Why are we patching this up twice? (It probably makes sense, I'm just not sure.)

Reply:
I don't publish it here, I merely _use_ it for testing purposes. Its OSGi metadata is wrong, and so I have no choice but to patch it.